java/Anton-Kozyk/src/com/zoolatech/lecture3/tasks/_2/MainValidator.java

```diff
@@ -0,0 +1,31 @@
+package com.zoolatech.lecture3.tasks._2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MainValidator implements Validating {
+    private List<Validator> validators = new ArrayList<>();
+    private boolean validationResult = true;
+
+    @Override
+    public boolean isValid(UserAccount account) {
+        validators.add(new StringIsNotEmptyValidator("First name"));
```

Comment:
It would be better to add these validators during the object construction. Right now, new validator instances will be added to the list on each call of the `isValid` method